### PR TITLE
Test order resources

### PR DESCRIPTION
**Pull request by Guimino**
Created Services to retrieve Test Concepts and its category of Laboratorio Geral do be used to  create the Laboratoriy Test requesitions, also create the  rest web service resources.

Created the Services to Create/update/delete the  Laboratory Test Requisitions, also created the rest web services resources.

Created the services to create/update/delete the Laboratory Test Results (for the laboratory requested tests) , also created the rest web services resources

### CODE REVIEW

**Comment by Guimino** [comment]
[9.1.1. Develop component to capture laboratory tests ordered during a clinical visit](https://trello.com/c/EGZLBOPs/33-911-develop-component-to-capture-laboratory-tests-ordered-during-a-clinical-visit)

**Comment by ynurmahomed** [comment]
[9.2.1. Develop a module to allow the entry of lab test results through the POC interface](https://trello.com/c/ElOKAERd/31-921-develop-a-module-to-allow-the-entry-of-lab-test-results-through-the-poc-interface)

**Comment by valvijo** [comment]
@ningosi is there any workaround to get the dependencies needed here?

File: api/src/main/java/org/openmrs/module/poc/api/common/util/OPENMRSUUIDs.java [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.poc.api.common.util;
+
+public interface OPENMRSUUIDs {
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/poc/api/common/util/OPENMRSUUIDs.java]: change this from interface to class and all constants should be public static final. This an example of how openmrs does and we would like to maintain those 
**public static final String APPLICATION_DATA_DIRECTORY_RUNTIME_PROPERTY = "application_data_directory";**
When calling them you just use class and the property

Comment (ningosi) [reply api/src/main/java/org/openmrs/module/poc/api/common/util/OPENMRSUUIDs.java]: class name could be
public class PocConstants{
public static final String DEFAULT_ENCOUNTER_ROLE_UUID = "a0b03050-c99b-11e0-9572-0800200c9a66";
.
.
.
}

Comment (ningosi) [reply api/src/main/java/org/openmrs/module/poc/api/common/util/OPENMRSUUIDs.java]: in openmrs that is refereed to as shown below
https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/EncounterRole.java#L20

File: api/src/main/java/org/openmrs/module/poc/pocheuristic/dao/PocHeuristicCAO.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import java.util.Date;
+import java.util.List;
+
+import org.hibernate.SessionFactory;
+import org.openmrs.Encounter;
+import org.openmrs.EncounterType;
+import org.openmrs.Location;
+import org.openmrs.Patient;
+
+public interface PocHeuristicCAO {
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/poc/pocheuristic/dao/PocHeuristicCAO.java]: was this suppossed to be **PocHeuristicDAO** as oppossed to **PocHeuristicCAO**?

**Comment by ningosi** [comment]
I will merge this work for testing, pending coding style/standards improvemnts

**Comment by ynurmahomed** [comment]
[Develop OpenMRS module](https://trello.com/c/UGVLBv7L/56-develop-openmrs-module)

**Pull request merged**